build/Dockerfile.standalone-tests

```diff
@@ -0,0 +1,20 @@
+# EKS (amazonlinux) version:
+# --build-arg FDBVERSION=7.1.3-${OKTETO_USER}-debug \
+# --build-arg FDBREPO=${AWS_ACCOUNT_ID}.dkr.ecr.${AWS_REGION}.amazonaws.com/foundationdb \
+# --build-arg JDKPKG=java-11-amazon-corretto-11.0.13+8-1.amzn2
+
+ARG FDBVERSION=7.1.3
+ARG FDBREPO=foundationdb
+
+FROM ${FDBREPO}/foundationdb-base:${FDBVERSION} AS java
+
+ARG JDKPKG=java-11-openjdk-11.0.13.0.8-1.el7_9
+
+RUN yum -y install ${JDKPKG} && yum clean all && rm -rf /var/cache/yum
+
+FROM java AS standalone
+
+COPY fdb-record-layer-core/.out/libs/fdb-record-layer-core-3.1-SNAPSHOT-standalone-tests.jar fdb-record-layer-standalone-tests.jar 
```

Comment:
Isn't that hard in docker files?

Reply:
Sorry: I meant the `RECORD_LAYER_VERSION`:
`COPY fdb-record-layer-core/.out/libs/fdb-record-layer-core-${RECORD_LAYER_VERSION}-SNAPSHOT-standalone-tests.jar fdb-record-layer-standalone-tests.jar`
Just so that the record layer version and the fdb version are both `ARG`s at the top.

Reply:
Taking that up in another PR.